azure-iot-device/samples/pnp/pnp_methods.py

```diff
@@ -0,0 +1,111 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module knows how to convert device SDK functionality into a PNP functionality.
+These methods formats the telemetry, methods, properties to PNP relevant telemetry,
+command requests and pnp properties.
+"""
+from azure.iot.device import Message, MethodResponse
+import json
+from pnp_helper import retrieve_values_dict_from_payload, create_command_response_payload
```

Comment:
When importing a module with functions rather than classes, I think it probably best to just import the module and then access the functions via the module.

e.g. import pnp_helper
...

pnp_helper.retrieve_values_dict_from_payload(...)